src/app/splashscreen/splashscreen.component.scss

```diff
@@ -0,0 +1,5 @@
+@import "../styles/components/headings";
+
+.logo {
+    max-width: 25%;
```

Comment:
better to resize the image (unless the same image is used elsewhere), to have less payload on the server